plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -1058,7 +1142,7 @@ int create_log_group(struct flb_cloudwatch *ctx)
             ctx->group_created = FLB_TRUE;
             flb_sds_destroy(body);
             flb_http_client_destroy(c);
-            return 0;
+            return set_log_group_retention(ctx);
```

Comment:
nit: Eduardo has told me that in C it is not good style to return a function call. So instead:

```
ret = set_log_group_retention(ctx);
return ret;
```